twilio cli inside a docker container - issues with npm pkg window-size

Hello! 

I'm running the Twilio-cli inside a docker container, running "twilio server less:start" works perfectly if I run it manually from inside the running container but it fails when run using the docker CMD in the Dockerfile or from the entrypoint file. I didn't dig too much but it looks like the window-size npm pkg doesn't like to run without an actual console.

Here's the error I get:

```
twilio    | > twilio serverless:start
twilio    |
twilio    | TypeError: Cannot read property 'width' of undefined
twilio    |     at wrapText (~/.twilio-cli/node_modules/twilio-run/dist/printers/utils.js:32:76)
twilio    |     at importantMessage (~/.twilio-cli/node_modules/twilio-run/dist/printers/utils.js:35:13)
twilio    |     at Object.warningMessage (~/.twilio-cli/node_modules/twilio-run/dist/printers/utils.js:41:12)
twilio    |     at Logger.warn (~/.twilio-cli/node_modules/twilio-run/dist/utils/logger.js:49:23)
twilio    |     at printVersionWarning (~/.twilio-cli/node_modules/twilio-run/dist/checks/nodejs-version.js:16:21)
twilio    |     at Object.checkNodejsVersion [as default] (~/.twilio-cli/node_modules/twilio-run/dist/checks/nodejs-version.js:22:9)
twilio    |     at ~/.twilio-cli/node_modules/twilio-run/dist/commands/start.js:27:33
twilio    |     at Generator.next (<anonymous>)
twilio    |     at ~/.twilio-cli/node_modules/twilio-run/dist/commands/start.js:7:71
twilio    |     at new Promise (<anonymous>)
twilio    | npm ERR! code ELIFECYCLE
twilio    | npm ERR! errno 1
twilio    | npm ERR! example@0.0.0 serverless-start: `twilio serverless:start`
```



```
~/.twilio-cli/node_modules/twilio-run/dist/printers/utils.js:32:76
---

const window_size_1 = __importDefault(require("window-size"));
..

###### I would perform a quick check on "window_size_1" and maybe use a default value ? ######
const wrapText = (text) => wrap_ansi_1.default(text, window_size_1.default.width - 5, { trim: false });
function importantMessage(label, color, title, body) {
    label = chalk_1.default.keyword(color)(label);
    title = wrapText(chalk_1.default.bold.underline(`${label} ${chalk_1.default.bold(title)}`));
    body = wrapText(body);
    return '\n' + borderLeft(`${title}\n\n${chalk_1.default.dim(body)}`, color) + '\n';
}
```

--- a/src/printers/list.ts
+++ b/src/printers/list.ts
@@ -14,17 +14,16 @@ import { stripIndent } from 'common-tags';
 import startCase from 'lodash.startcase';
 import logSymbols from 'log-symbols';
 import title from 'title';
-import size from 'window-size';
 import { ListConfig } from '../config/list';
 import { logger } from '../utils/logger';
 import { writeOutput } from '../utils/output';
-import { redactPartOfString, shouldPrettyPrint } from './utils';
+import { redactPartOfString, shouldPrettyPrint, windowSize } from './utils';
 
 type KeyMaps = {
   [key in ListOptions]: string[];
 };
 
-const LONG_LINE = '─'.repeat(size.width);
+const LONG_LINE = '─'.repeat(windowSize.width);
 
 const baseKeys: KeyMaps = {
   environments: [

--- a/src/printers/start.ts
+++ b/src/printers/start.ts
@@ -2,12 +2,11 @@ import { ServerlessResourceConfig } from '@twilio-labs/serverless-api';
 import boxen from 'boxen';
 import chalk from 'chalk';
 import logSymbols from 'log-symbols';
-import windowSize from 'window-size';
 import wrapAnsi from 'wrap-ansi';
 import { StartCliConfig } from '../config/start';
 import { getFunctionsAndAssets } from '../runtime/internal/runtime-paths';
 import { logger } from '../utils/logger';
-import { shouldPrettyPrint, terminalLink } from './utils';
+import { shouldPrettyPrint, terminalLink, windowSize } from './utils';
 
 function printAsset(
   asset: ServerlessResourceConfig,

--- a/src/printers/utils.ts
+++ b/src/printers/utils.ts
@@ -4,6 +4,7 @@ import isCi from 'is-ci';
 import size from 'window-size';
 import wrapAnsi from 'wrap-ansi';
 
+export const windowSize = size || { width: 80, height: 300 };
 export const shouldPrettyPrint = process.stdout.isTTY && !isCi;
 export const supportsEmoji =
   process.platform !== 'win32' || isCi || process.env.TERM === 'xterm-256color';
@@ -31,7 +32,7 @@ export function borderLeft(text: string, color: string): string {
 }
 
 const wrapText = (text: string) =>
-  wrapAnsi(text, size.width - 5, { trim: false });
+  wrapAnsi(text, windowSize.width - 5, { trim: false });
 
 export function importantMessage(
   label: string,

